### ISSUE TITLE
Error beim Verbinden von Redcap mit LDAP

### ISSUE THREAD

**Issue by jkiederle** [medizininformatik-initiative/INTERPOLAR#488]
Hallo,

ich möchte das Redcap an das Klinik LDAP anschließen, damit unsere Apotheker:innen ihr Klinik-login benutzen können. Aktuell ist es Table-based, und wenn ich es ändere zu Table based&LDAP und dann den Trouble-shooter für LDAP für REDcap nutze sehe ich diesen Fehler: Auth_Container_LDAP: LDAP Extension not loaded
Ich habe bereits im Dockerfile von der Redcap-app das hinzugefügt:

```
RUN apt-get install php8.0-ldap \
 
RUN docker-php-ext-configure ldap \
    && docker-php-ext-install ldap \
    && docker-php-ext-enable  ldap \
```

(zuvor hatte ich auch beides einzeln hinzugefügt). Leider ohne Erfolg.

Außerdem hatte ich in dem file php.ini diese Zeile hinzugefügt:

`add extension=ldap.so`

Leider gab es immer die gleiche Fehlermeldung.

Als ich im Docker Container von der Redcap-app war und php -m aufgerufen habe, habe ich auch das ldap package nicht gefunden.

Ich würde mich freuen, wenn jemand mir dabei helfen könnte! 

Viele Grüße,
Jana

**Comment by SebStaeubert** [comment]
Konnte das Problem mittlerweilse gelöst werden?

**Comment by jkiederle** [comment]
Nein, mir konnte weder jemand am Standort noch von eurer Seite helfen, also habe ich Redcap mit Table based usern eingerichtet. Das geht ja auch gut :)
